src/quo2/components/drawers/action_drawers.cljs

```diff
@@ -65,10 +17,60 @@
                     :align-items      :center
                     :flex-direction   :row}}])
 
+(defn action [{:keys [icon
+                      label
+                      sub-label
+                      right-icon
+                      danger?
+                      on-press
+                      add-divider?] :as action-props}]
+  (when action-props
+    [:<> {:key label}
+     (when add-divider? [divider])
+     [rn/touchable-opacity {:on-press on-press}
+      [rn/view {:style
+                {:height            (if sub-label 56 47)
+                 :margin-horizontal 20
+                 :flex-direction    :row}}
+       [rn/view {:style
+                 {:height        20
+                  :margin-top    :auto
+                  :margin-bottom :auto
+                  :margin-right  12
+                  :width         20}}
+        [icon/icon icon
+         {:color (get-icon-color danger?)
+          :size  20}]]
+       [rn/view
+        {:style
+         {:flex            1
+          :justify-content :center}}
+        [text/text
+         {:size   :paragraph-1
+          :weight :medium
+          :style  {:color
+                   (when danger?
+                     (colors/theme-colors colors/danger-50 colors/danger-60))}}
+         label]
+        (when sub-label
+          [text/text
+           {:size  :paragraph-2
+            :style {:color
+                    (colors/theme-colors colors/neutral-50 colors/neutral-40)}}
+           sub-label])]
+       (when right-icon
+         [rn/view {:style
+                   {:height        20
+                    :margin-top    :auto
+                    :margin-bottom :auto
+                    :width         20}}
+          [icon/icon right-icon
+           {:color (get-icon-color danger?)
+            :size  20}]])]]]))
+
 (defn action-drawer [sections]
-  [:<> {:style
-        {:flex 1}}
-   (interpose
-    [divider]
+  [:<>
+   (doall
     (for [actions sections]
-      (map action actions)))])
+      (doall
```

Comment:
are you sure its needed twice ?

Reply:
Yes

Reply:
And tbh I do not know what does `doall` do, but the warning was telling me to wrap it with `doall`. Removing either of the `doall` brings back the warning.